dapr-spring/dapr-spring-boot-testcontainers/src/main/java/io/dapr/testcontainers/DaprModule.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
@artur-ciocanu are we using this anywhere downstream? maybe we just don't need this, the entire module is just an example and I can see why people will never use this.

Reply:
I have moved this to `sdk-tests` to keep it as an example.